From a4384801367c241b27e263c085344b0436d83905 Mon Sep 17 00:00:00 2001
From: Raj Rajasekaran <b10872@freescale.com>
Date: Sat, 1 Nov 2008 01:40:09 -0500
Subject: [PATCH 028/249] ENGR00097414 Sahara: Lib makefile should build
 Sahara for platforms that has it.

Added a rule to build Sahara library only for platforms which support it.

Signed-off-by: Raj Rajasekaran <b10872@freescale.com>
---
 sahara2/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sahara2/Makefile b/sahara2/Makefile
index fd60f4f..a5a2dfc 100644
--- a/sahara2/Makefile
+++ b/sahara2/Makefile
@@ -43,9 +43,9 @@ CFLAGS += $(TARGET_ARCH)
 
 LIBNAME=libsahara
 
-#ifeq ("$(TARGET_ARCH)", "")
-#all install:
-#else
+ifeq ("$(TARGET_ARCH)", "")
+all install:
+else
 all: $(LIBNAME).so $(LIBNAME).a
 
 install: install_headers
@@ -54,7 +54,7 @@ install: install_headers
 
 install_headers:
 	@mkdir -p $(DEST_DIR)/usr/include
-#endif
+endif
 
 
 $(LIBNAME).a: $(OBJS)
-- 
1.8.0

